performant-software/DM
# Remove highlight blockers and selections on text resource load

### What does this PR do?
Removes the "no-highlight" and "selected" classes from annotation elements in a text resource during load

### What issues does this address?
#99, #97 

### How to test
- See description of inconsistent highlighting in #97 and in Martin's video
- On staging (http://67.207.158.85/), open text resources and set them to various different states (highlights toggled off, highlights toggled on, only one highlight showing after opening resource from annotation list), then reload the page, reopen the same resources, and test their highlight display features
